astropy/io/fits/connect.py

```diff
@@ -173,6 +177,11 @@ def read_table_fits(input, hdu=None):
                 t.meta[key].append(value)
             else:
                 t.meta[key] = [value]
+            if 'AstroPy' in value:
```

Comment:
This should probably be removed entirely, but even if it remains the test needs to be more stringent.  It's not impossible that a random application writes a FITS file with a comment that includes `AstroPy` somewhere.